packages/calling-stateful-client/src/StreamUtils.ts

```diff
@@ -34,21 +35,27 @@ async function createViewRemoteVideo(
   } else {
     participantKey = toFlatCommunicationIdentifier(participantId);
   }
+  const streamLogInfo = `callId: ${callId}, participantKey: ${participantKey}, streamId: ${streamId}`;
+
+  callingStatefulLogger.info(`Start creating view for remote video! ${streamLogInfo}`);
```

Comment:
From @alkwa-msft 's comment on not impacting network bandwidth during calls -- when is this telemetry sent? Is there a way to batch-and-wait ?

Reply:
This will only output things into console if you call logger.setLevel("info"), it wouldn't do anything without anyone call this line
And it will only be redirected into app insight once contoso app calls the connector api(currently only in selfhost app, and network bandwidth friendly implementation will be implemented there)

Reply:
Nice!

Reply:
Can you add something in our Docs > References somewhere that mentions this -- it's really hard to find info online about the logger

Reply:
Sure, we will have more docs around this, and we've used this a lot already in state change. Before we check in any telemetry api into this repo, let's treat this as an internal conditional console.log because this is how it behaves for now.
And the behavior will only change once we have the logger connector api, and that would be the time we add docs, currently it would be a little confusing if we add anything (cause no api or implementation is provided)